In [ ]:

# IMPORTANT: RUN THIS CELL IN ORDER TO IMPORT YOUR KAGGLE DATA SOURCES
# TO THE CORRECT LOCATION (/kaggle/input) IN YOUR NOTEBOOK,
# THEN FEEL FREE TO DELETE THIS CELL.
# NOTE: THIS NOTEBOOK ENVIRONMENT DIFFERS FROM KAGGLE'S PYTHON
# ENVIRONMENT SO THERE MAY BE MISSING LIBRARIES USED BY YOUR
# NOTEBOOK.

import os
import sys
from tempfile import NamedTemporaryFile
from urllib.request import urlopen
from urllib.parse import unquote, urlparse
from urllib.error import HTTPError
from zipfile import ZipFile
import tarfile
import shutil

CHUNK_SIZE = 40960
DATA_SOURCE_MAPPING = 'titanic:https%3A%2F%2Fstorage.googleapis.com%2Fkaggle-competitions-data%2Fkaggle-v2%2F3136%2F26502%2Fbundle%2Farchive.zip%3FX-Goog-Algorithm%3DGOOG4-RSA-SHA256%26X-Goog-Credential%3Dgcp-kaggle-com%2540kaggle-161607.iam.gserviceaccount.com%252F20240124%252Fauto%252Fstorage%252Fgoog4_request%26X-Goog-Date%3D20240124T124449Z%26X-Goog-Expires%3D259200%26X-Goog-SignedHeaders%3Dhost%26X-Goog-Signature%3D02bd8edd546fbf5976d1da075acee7987859e17c8e9ab3cefa52cc51581812089e29aa24fb637a9dcc34b12a96b4fd214393dff06fcab74952e01568d5faf3bdcd5524730423f489add567cb62407d812aaebf01185c905bd7fb1ff02e3bdd0cf29d4e3e297d371766de3c5edd2ab77f4458901cd3e1cdb34442f66c93d09492c87fdc0382557331a0e79aa8809c47fc3aa64c73c17e53889f8522876affb691fea6ea20f6698d59233fc39630707963d459946af801b5477ccf89e94983c7a2d6ce4cae7823874287c0d1eb5f93b99c8926fbca74082dcdd4ca5d7fb7b504707f5b11f73a0ba5f59ec5c93c6cbf2a21fc967d018d2b092b710a08a315b0ed01'

KAGGLE_INPUT_PATH='/kaggle/input'
KAGGLE_WORKING_PATH='/kaggle/working'
KAGGLE_SYMLINK='kaggle'

!umount /kaggle/input/ 2> /dev/null
shutil.rmtree('/kaggle/input', ignore_errors=True)
os.makedirs(KAGGLE_INPUT_PATH, 0o777, exist_ok=True)
os.makedirs(KAGGLE_WORKING_PATH, 0o777, exist_ok=True)

try:
  os.symlink(KAGGLE_INPUT_PATH, os.path.join("..", 'input'), target_is_directory=True)
except FileExistsError:
  pass
try:
  os.symlink(KAGGLE_WORKING_PATH, os.path.join("..", 'working'), target_is_directory=True)
except FileExistsError:
  pass

for data_source_mapping in DATA_SOURCE_MAPPING.split(','):
    directory, download_url_encoded = data_source_mapping.split(':')
    download_url = unquote(download_url_encoded)
    filename = urlparse(download_url).path
    destination_path = os.path.join(KAGGLE_INPUT_PATH, directory)
    try:
        with urlopen(download_url) as fileres, NamedTemporaryFile() as tfile:
            total_length = fileres.headers['content-length']
            print(f'Downloading {directory}, {total_length} bytes compressed')
            dl = 0
            data = fileres.read(CHUNK_SIZE)
            while len(data) > 0:
                dl += len(data)
                tfile.write(data)
                done = int(50 * dl / int(total_length))
                sys.stdout.write(f"\r[{'=' * done}{' ' * (50-done)}] {dl} bytes downloaded")
                sys.stdout.flush()
                data = fileres.read(CHUNK_SIZE)
            if filename.endswith('.zip'):
              with ZipFile(tfile) as zfile:
                zfile.extractall(destination_path)
            else:
              with tarfile.open(tfile.name) as tarfile:
                tarfile.extractall(destination_path)
            print(f'\nDownloaded and uncompressed: {directory}')
    except HTTPError as e:
        print(f'Failed to load (likely expired) {download_url} to path {destination_path}')
        continue
    except OSError as e:
        print(f'Failed to load {download_url} to path {destination_path}')
        continue

print('Data source import complete.')


# Titanic competition with TensorFlow Decision Forests

This notebook will take you through the steps needed to train a baseline Gradient Boosted Trees Model using TensorFlow Decision Forests and creating a submission on the Titanic competition.

This notebook shows:

1. How to do some basic pre-processing. For example, the passenger names will be tokenized, and ticket names will be splitted in parts.
1. How to train a Gradient Boosted Trees (GBT) with default parameters
1. How to train a GBT with improved default parameters
1. How to tune the parameters of a GBTs
1. How to train and ensemble many GBTs

# Imports dependencies

In [3]:
import numpy as np
import pandas as pd
import os

import tensorflow as tf
# !pip install tensorflow_decision_forests
import tensorflow_decision_forests as tfdf

print(f"Found TF-DF {tfdf.__version__}")

Found TF-DF 1.8.1


# Load dataset

In [4]:
train_df = pd.read_csv("data/train.csv")
serving_df = pd.read_csv("data/test.csv")

train_df.head(10)

PassengerId  Survived  Pclass  \
0            1         0       3   
1            2         1       1   
2            3         1       3   
3            4         1       1   
4            5         0       3   
5            6         0       3   
6            7         0       1   
7            8         0       3   
8            9         1       3   
9           10         1       2   

                                                Name     Sex   Age  SibSp  \
0                            Braund, Mr. Owen Harris    male  22.0      1   
1  Cumings, Mrs. John Bradley (Florence Briggs Th...  female  38.0      1   
2                             Heikkinen, Miss. Laina  female  26.0      0   
3       Futrelle, Mrs. Jacques Heath (Lily May Peel)  female  35.0      1   
4                           Allen, Mr. William Henry    male  35.0      0   
5                                   Moran, Mr. James    male   NaN      0   
6                            McCarthy, Mr. Timothy J    male  54.0      0   
7                     Palsson, Master. Gosta Leonard    male   2.0      3   
8  Johnson, Mrs. Oscar W (Elisabeth Vilhelmina Berg)  female  27.0      0   
9                Nasser, Mrs. Nicholas (Adele Achem)  female  14.0      1   

   Parch            Ticket     Fare Cabin Embarked  
0      0         A/5 21171   7.2500   NaN        S  
1      0          PC 17599  71.2833   C85        C  
2      0  STON/O2. 3101282   7.9250   NaN        S  
3      0            113803  53.1000  C123        S  
4      0            373450   8.0500   NaN        S  
5      0            330877   8.4583   NaN        Q  
6      0             17463  51.8625   E46        S  
7      1            349909  21.0750   NaN        S  
8      2            347742  11.1333   NaN        S  
9      0            237736  30.0708   NaN        C

# Prepare dataset

We will apply the following transformations on the dataset.

1. Tokenize the names. For example, "Braund, Mr. Owen Harris" will become ["Braund", "Mr.", "Owen", "Harris"].
2. Extract any prefix in the ticket. For example ticket "STON/O2. 3101282" will become "STON/O2." and 3101282.

In [5]:
def preprocess(df):
    df = df.copy()

    def normalize_name(x):
        return " ".join([v.strip(",()[].\"'") for v in x.split(" ")])

    def ticket_number(x):
        return x.split(" ")[-1]

    def ticket_item(x):
        items = x.split(" ")
        if len(items) == 1:
            return "NONE"
        return "_".join(items[0:-1])

    df["Name"] = df["Name"].apply(normalize_name)
    df["Ticket_number"] = df["Ticket"].apply(ticket_number)
    df["Ticket_item"] = df["Ticket"].apply(ticket_item)
    return df

preprocessed_train_df = preprocess(train_df)
preprocessed_serving_df = preprocess(serving_df)

preprocessed_train_df.head(5)

PassengerId  Survived  Pclass  \
0            1         0       3   
1            2         1       1   
2            3         1       3   
3            4         1       1   
4            5         0       3   

                                              Name     Sex   Age  SibSp  \
0                            Braund Mr Owen Harris    male  22.0      1   
1  Cumings Mrs John Bradley Florence Briggs Thayer  female  38.0      1   
2                             Heikkinen Miss Laina  female  26.0      0   
3         Futrelle Mrs Jacques Heath Lily May Peel  female  35.0      1   
4                           Allen Mr William Henry    male  35.0      0   

   Parch            Ticket     Fare Cabin Embarked Ticket_number Ticket_item  
0      0         A/5 21171   7.2500   NaN        S         21171         A/5  
1      0          PC 17599  71.2833   C85        C         17599          PC  
2      0  STON/O2. 3101282   7.9250   NaN        S       3101282    STON/O2.  
3      0            113803  53.1000  C123        S        113803        NONE  
4      0            373450   8.0500   NaN        S        373450        NONE

Let's keep the list of the input features of the model. Notably, we don't want to train our model on the "PassengerId" and "Ticket" features.

In [6]:
input_features = list(preprocessed_train_df.columns)
input_features.remove("Ticket")
input_features.remove("PassengerId")
input_features.remove("Survived")
#input_features.remove("Ticket_number")

print(f"Input features: {input_features}")

Input features: ['Pclass', 'Name', 'Sex', 'Age', 'SibSp', 'Parch', 'Fare', 'Cabin', 'Embarked', 'Ticket_number', 'Ticket_item']


# Convert Pandas dataset to TensorFlow Dataset

In [7]:
def tokenize_names(features, labels=None):
    """Divite the names into tokens. TF-DF can consume text tokens natively."""
    features["Name"] =  tf.strings.split(features["Name"])
    return features, labels

train_ds = tfdf.keras.pd_dataframe_to_tf_dataset(preprocessed_train_df,label="Survived").map(tokenize_names)
serving_ds = tfdf.keras.pd_dataframe_to_tf_dataset(preprocessed_serving_df).map(tokenize_names)

# Train model with default parameters

### Train model

First, we are training a GradientBoostedTreesModel model with the default parameters.

In [8]:
model = tfdf.keras.GradientBoostedTreesModel(
    verbose=0, # Very few logs
    features=[tfdf.keras.FeatureUsage(name=n) for n in input_features],
    exclude_non_specified_features=True, # Only use the features in "features"
    random_seed=1234,
)
model.fit(train_ds)

self_evaluation = model.make_inspector().evaluation()
print(f"Accuracy: {self_evaluation.accuracy} Loss:{self_evaluation.loss}")

[WARNING 24-01-24 17:00:23.1870 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:00:23.1872 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:00:23.1872 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:00:31.5195 +04 kernel.cc:1233] Loading model from path /tmp/tmpv44fz1c5/model/ with prefix fdff4cd81d074d7e
[INFO 24-01-24 17:00:31.5252 +04 quick_scorer_extended.cc:903] The binary was compiled without AVX2 support, but your CPU supports it. Enable it for faster model inference.
[INFO 24-01-24 17:00:31.5279 +04 abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-01-24 17:00:31.5287 +04 kernel.cc:1061] Use fast generic engine


Accuracy: 0.8260869383811951 Loss:0.8608942627906799


# Train model with improved default parameters

Now you'll use some specific parameters when creating the GBT model

In [9]:
model = tfdf.keras.GradientBoostedTreesModel(
    verbose=0, # Very few logs
    features=[tfdf.keras.FeatureUsage(name=n) for n in input_features],
    exclude_non_specified_features=True, # Only use the features in "features"

    #num_trees=2000,

    # Only for GBT.
    # A bit slower, but great to understand the model.
    # compute_permutation_variable_importance=True,

    # Change the default hyper-parameters
    # hyperparameter_template="benchmark_rank1@v1",

    #num_trees=1000,
    #tuner=tuner

    min_examples=1,
    categorical_algorithm="RANDOM",
    #max_depth=4,
    shrinkage=0.05,
    #num_candidate_attributes_ratio=0.2,
    split_axis="SPARSE_OBLIQUE",
    sparse_oblique_normalization="MIN_MAX",
    sparse_oblique_num_projections_exponent=2.0,
    num_trees=2000,
    #validation_ratio=0.0,
    random_seed=1234,

)
model.fit(train_ds)

self_evaluation = model.make_inspector().evaluation()
print(f"Accuracy: {self_evaluation.accuracy} Loss:{self_evaluation.loss}")

[WARNING 24-01-24 17:01:58.8151 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:01:58.8152 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:01:58.8152 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:02:00.8697 +04 kernel.cc:1233] Loading model from path /tmp/tmp07teiglr/model/ with prefix 7da856bd0fae40e0
[INFO 24-01-24 17:02:00.8743 +04 decision_forest.cc:660] Model loaded with 40 root(s), 2106 node(s), and 10 input feature(s).
[INFO 24-01-24 17:02:00.8743 +04 abstract_model.cc:1344] Engine "GradientBoostedTreesGeneric" built
[INFO 24-01-24 17:02:00.8743 +04 kernel.cc:1061] Use fast generic engine


Accuracy: 0.782608687877655 Loss:1.0586705207824707


Let's look at the model and you can also notice the information about variable importance that the model figured out

In [10]:
model.summary()

Model: "gradient_boosted_trees_model_1"
_________________________________________________________________
 Layer (type)                Output Shape              Param #   
Total params: 1 (1.00 Byte)
Trainable params: 0 (0.00 Byte)
Non-trainable params: 1 (1.00 Byte)
_________________________________________________________________
Type: "GRADIENT_BOOSTED_TREES"
Task: CLASSIFICATION
Label: "__LABEL"

Input Features (11):
	Age
	Cabin
	Embarked
	Fare
	Name
	Parch
	Pclass
	Sex
	SibSp
	Ticket_item
	Ticket_number

No weights

Variable Importance: INV_MEAN_MIN_DEPTH:
    1.           "Sex"  0.585997 ################
    2.           "Age"  0.364636 #######
    3.          "Fare"  0.266191 ###
    4.          "Name"  0.207054 #
    5.        "Pclass"  0.179191 
    6. "Ticket_number"  0.178806 
    7.      "Embarked"  0.177803 
    8.   "Ticket_item"  0.177009 
    9.         "Parch"  0.175276 
   10.         "SibSp"  0.171694 

Variable Importance: NUM_AS_ROOT:
    1.  "Sex" 34.000000 ######

# Make predictions

In [14]:
def prediction_to_kaggle_format(model, threshold=0.5):
    proba_survive = model.predict(serving_ds, verbose=0)[:,0]
    return pd.DataFrame({
        "PassengerId": serving_df["PassengerId"],
        "Survived": (proba_survive >= threshold).astype(int)
    })

def make_submission(kaggle_predictions):
    path="data/tf_lead_submission.csv"
    kaggle_predictions.to_csv(path, index=False)
    print(f"Submission exported to {path}")

kaggle_predictions = prediction_to_kaggle_format(model)
make_submission(kaggle_predictions)
!head data/tf_lead_submission.csv

Submission exported to data/tf_lead_submission.csv
PassengerId,Survived
892,0
893,0
894,0
895,0
896,0
897,0
898,0
899,0
900,1


# Training a model with hyperparameter tunning

Hyper-parameter tuning is enabled by specifying the tuner constructor argument of the model. The tuner object contains all the configuration of the tuner (search space, optimizer, trial and objective).


In [15]:
tuner = tfdf.tuner.RandomSearch(num_trials=1000)
tuner.choice("min_examples", [2, 5, 7, 10])
tuner.choice("categorical_algorithm", ["CART", "RANDOM"])

local_search_space = tuner.choice("growing_strategy", ["LOCAL"])
local_search_space.choice("max_depth", [3, 4, 5, 6, 8])

global_search_space = tuner.choice("growing_strategy", ["BEST_FIRST_GLOBAL"], merge=True)
global_search_space.choice("max_num_nodes", [16, 32, 64, 128, 256])

#tuner.choice("use_hessian_gain", [True, False])
tuner.choice("shrinkage", [0.02, 0.05, 0.10, 0.15])
tuner.choice("num_candidate_attributes_ratio", [0.2, 0.5, 0.9, 1.0])


tuner.choice("split_axis", ["AXIS_ALIGNED"])
oblique_space = tuner.choice("split_axis", ["SPARSE_OBLIQUE"], merge=True)
oblique_space.choice("sparse_oblique_normalization",
                     ["NONE", "STANDARD_DEVIATION", "MIN_MAX"])
oblique_space.choice("sparse_oblique_weights", ["BINARY", "CONTINUOUS"])
oblique_space.choice("sparse_oblique_num_projections_exponent", [1.0, 1.5])

# Tune the model. Notice the `tuner=tuner`.
tuned_model = tfdf.keras.GradientBoostedTreesModel(tuner=tuner)
tuned_model.fit(train_ds, verbose=0)

tuned_self_evaluation = tuned_model.make_inspector().evaluation()
print(f"Accuracy: {tuned_self_evaluation.accuracy} Loss:{tuned_self_evaluation.loss}")

Use /tmp/tmpmo01405c as temporary training directory


[WARNING 24-01-24 17:04:22.0598 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:04:22.0599 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:04:22.0599 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:06:14.4019 +04 kernel.cc:1233] Loading model from path /tmp/tmpmo01405c/model/ with prefix 0f4f4a7c01f3441e
[INFO 24-01-24 17:06:14.4293 +04 decision_forest.cc:660] Model loaded with 30 root(s), 2236 node(s), and 12 input feature(s).
[INFO 24-01-24 17:06:14.4293 +04 abstract_model.cc:1344] Engine "GradientBoostedTreesGeneric" built
[INFO 24-01-24 17:06:14.4294 +04 kernel.cc:1061] Use fast generic engine


Accuracy: 0.8630136847496033 Loss:0.6749962568283081


In the last line in the cell above, you can see the accuracy is higher than previously with default parameters and parameters set by hand.

This is the main idea behing hyperparameter tuning.

For more information you can follow this tutorial: [Automated hyper-parameter tuning](https://www.tensorflow.org/decision_forests/tutorials/automatic_tuning_colab)

# Making an ensemble

Here you'll create 100 models with different seeds and combine their results

This approach removes a little bit the random aspects related to creating ML models

In the GBT creation is used the `honest` parameter. It will use different training examples to infer the structure and the leaf values. This regularization technique trades examples for bias estimates.

In [16]:
predictions = None
num_predictions = 0

for i in range(100):
    print(f"i:{i}")
    # Possible models: GradientBoostedTreesModel or RandomForestModel
    model = tfdf.keras.GradientBoostedTreesModel(
        verbose=0, # Very few logs
        features=[tfdf.keras.FeatureUsage(name=n) for n in input_features],
        exclude_non_specified_features=True, # Only use the features in "features"

        #min_examples=1,
        #categorical_algorithm="RANDOM",
        ##max_depth=4,
        #shrinkage=0.05,
        ##num_candidate_attributes_ratio=0.2,
        #split_axis="SPARSE_OBLIQUE",
        #sparse_oblique_normalization="MIN_MAX",
        #sparse_oblique_num_projections_exponent=2.0,
        #num_trees=2000,
        ##validation_ratio=0.0,
        random_seed=i,
        honest=True,
    )
    model.fit(train_ds)

    sub_predictions = model.predict(serving_ds, verbose=0)[:,0]
    if predictions is None:
        predictions = sub_predictions
    else:
        predictions += sub_predictions
    num_predictions += 1

predictions/=num_predictions

kaggle_predictions = pd.DataFrame({
        "PassengerId": serving_df["PassengerId"],
        "Survived": (predictions >= 0.5).astype(int)
    })

make_submission(kaggle_predictions)

i:0


[WARNING 24-01-24 17:07:42.9098 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:07:42.9098 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:07:42.9098 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:07:43.7270 +04 kernel.cc:1233] Loading model from path /tmp/tmpdh389vno/model/ with prefix d5ba164a2fa54db6
[INFO 24-01-24 17:07:43.7307 +04 quick_scorer_extended.cc:903] The binary was compiled without AVX2 support, but your CPU supports it. Enable it for faster model inference.
[INFO 24-01-24 17:07:43.7312 +04 abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-01-24 17:07:43.7312 +04 kernel.cc:1061] Use fast generic engine


i:1


[WARNING 24-01-24 17:07:44.3899 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:07:44.3900 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:07:44.3900 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


[INFO 24-01-24 17:07:45.5232 +04 kernel.cc:1233] Loading model from path /tmp/tmp50c_wi2o/model/ with prefix aa6f9cd96f394d88
[INFO 24-01-24 17:07:45.5394 +04 kernel.cc:1061] Use fast generic engine


i:2


[WARNING 24-01-24 17:07:46.2705 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:07:46.2706 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:07:46.2706 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".


[INFO 24-01-24 17:07:47.1488 +04 kernel.cc:1233] Loading model from path /tmp/tmppt2but4b/model/ with prefix 8638e58c377a4351
[INFO 24-01-24 17:07:47.1548 +04 kernel.cc:1061] Use fast generic engine


i:3


[WARNING 24-01-24 17:07:49.7407 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:07:49.7408 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:07:49.7408 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:07:52.0449 +04 kernel.cc:1233] Loading model from path /tmp/tmp407qlao7/model/ with prefix 38f8d2609bfd4919
[INFO 24-01-24 17:07:52.0643 +04 kernel.cc:1061] Use fast generic engine


i:4


[WARNING 24-01-24 17:07:52.6777 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:07:52.6778 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:07:52.6778 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:07:53.2612 +04 kernel.cc:1233] Loading model from path /tmp/tmpxv3o3qwl/model/ with prefix 1f7b0eb2f4374077
[INFO 24-01-24 17:07:53.2646 +04 kernel.cc:1061] Use fast generic engine


i:5


[WARNING 24-01-24 17:07:53.7459 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:07:53.7459 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:07:53.7459 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:07:54.7315 +04 kernel.cc:1233] Loading model from path /tmp/tmpl6z94htu/model/ with prefix d3d2879fc339412e
[INFO 24-01-24 17:07:54.7364 +04 abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-01-24 17:07:54.7367 +04 kernel.cc:1061] Use fast generic engine


i:6


[WARNING 24-01-24 17:07:55.3768 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:07:55.3768 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:07:55.3768 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:07:55.9675 +04 kernel.cc:1233] Loading model from path /tmp/tmpvw0d2363/model/ with prefix f4167ceb347c45af
[INFO 24-01-24 17:07:55.9732 +04 kernel.cc:1061] Use fast generic engine


i:7


[WARNING 24-01-24 17:07:56.6379 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:07:56.6380 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:07:56.6380 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:07:58.2948 +04 kernel.cc:1233] Loading model from path /tmp/tmp42m14rfw/model/ with prefix a624ae311c9244f3
[INFO 24-01-24 17:07:58.3229 +04 kernel.cc:1061] Use fast generic engine


i:8


[WARNING 24-01-24 17:07:59.9603 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:07:59.9603 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:07:59.9603 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:08:01.9495 +04 kernel.cc:1233] Loading model from path /tmp/tmp0ek6mbuu/model/ with prefix 14445661ac88441b
[INFO 24-01-24 17:08:01.9655 +04 kernel.cc:1061] Use fast generic engine


i:9


[WARNING 24-01-24 17:08:02.4930 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:02.4930 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:02.4930 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:08:03.2374 +04 kernel.cc:1233] Loading model from path /tmp/tmpxjcz6lhx/model/ with prefix 8d8d97f59eb34923
[INFO 24-01-24 17:08:03.2453 +04 kernel.cc:1061] Use fast generic engine


i:10


[WARNING 24-01-24 17:08:03.7371 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:03.7371 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:03.7371 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:08:04.3748 +04 kernel.cc:1233] Loading model from path /tmp/tmp3fn6cr0x/model/ with prefix 2bddbae96cdc4268
[INFO 24-01-24 17:08:04.3776 +04 kernel.cc:1061] Use fast generic engine


i:11


[WARNING 24-01-24 17:08:04.8574 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:04.8575 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:04.8575 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:08:05.5610 +04 kernel.cc:1233] Loading model from path /tmp/tmpf2znx4pw/model/ with prefix 38050d7d7ccc44af
[INFO 24-01-24 17:08:05.5687 +04 abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-01-24 17:08:05.5688 +04 kernel.cc:1061] Use fast generic engine


i:12


[WARNING 24-01-24 17:08:06.0323 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:06.0323 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:06.0323 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:08:06.6411 +04 kernel.cc:1233] Loading model from path /tmp/tmp6l3j8n4k/model/ with prefix 8e7e727705c34cb7
[INFO 24-01-24 17:08:06.6435 +04 kernel.cc:1061] Use fast generic engine


i:13


[WARNING 24-01-24 17:08:07.1092 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:07.1093 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:07.1093 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:08:09.1228 +04 kernel.cc:1233] Loading model from path /tmp/tmpu868af9j/model/ with prefix e3f4802bae8545dc
[INFO 24-01-24 17:08:09.1440 +04 kernel.cc:1061] Use fast generic engine


i:14


[WARNING 24-01-24 17:08:10.4057 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:10.4059 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:10.4059 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:08:12.1917 +04 kernel.cc:1233] Loading model from path /tmp/tmpq3k8eywr/model/ with prefix bf3a60f5b1b24887
[INFO 24-01-24 17:08:12.1969 +04 kernel.cc:1061] Use fast generic engine


i:15


[WARNING 24-01-24 17:08:12.7512 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:12.7512 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:12.7512 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:08:13.5819 +04 kernel.cc:1233] Loading model from path /tmp/tmpxugqsesq/model/ with prefix b73ac0b44c924efc
[INFO 24-01-24 17:08:13.5859 +04 kernel.cc:1061] Use fast generic engine


i:16


[WARNING 24-01-24 17:08:14.3650 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:14.3650 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:14.3650 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:08:15.1202 +04 kernel.cc:1233] Loading model from path /tmp/tmpmnb8dl43/model/ with prefix fc346bdde77a4cdb
[INFO 24-01-24 17:08:15.1259 +04 quick_scorer_extended.cc:903] The binary was compiled without AVX2 support, but your CPU supports it. Enable it for faster model inference.
[INFO 24-01-24 17:08:15.1276 +04 kernel.cc:1061] Use fast generic engine


i:17


[WARNING 24-01-24 17:08:15.6357 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:15.6357 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:15.6357 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:08:16.3668 +04 kernel.cc:1233] Loading model from path /tmp/tmpd_7dpagc/model/ with prefix a03dd9c5c1c8441e
[INFO 24-01-24 17:08:16.3740 +04 abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-01-24 17:08:16.3740 +04 kernel.cc:1061] Use fast generic engine


i:18


[WARNING 24-01-24 17:08:16.8585 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:16.8585 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:16.8585 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:08:18.1671 +04 kernel.cc:1233] Loading model from path /tmp/tmp52yp8swl/model/ with prefix c07b45f80c534c5f
[INFO 24-01-24 17:08:18.1857 +04 kernel.cc:1061] Use fast generic engine


i:19


[WARNING 24-01-24 17:08:19.2098 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:19.2100 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:19.2100 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:08:21.1952 +04 kernel.cc:1233] Loading model from path /tmp/tmp19nw_ph5/model/ with prefix e3bebf0fc306409c
[INFO 24-01-24 17:08:21.2113 +04 kernel.cc:1061] Use fast generic engine


i:20


[WARNING 24-01-24 17:08:22.2367 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:22.2367 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:22.2367 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:08:23.0730 +04 kernel.cc:1233] Loading model from path /tmp/tmpspa19rfe/model/ with prefix e6dc7ecbdac44b40
[INFO 24-01-24 17:08:23.0820 +04 kernel.cc:1061] Use fast generic engine


i:21


[WARNING 24-01-24 17:08:23.6169 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:23.6169 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:23.6170 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:08:24.2371 +04 kernel.cc:1233] Loading model from path /tmp/tmpg4rheh84/model/ with prefix 34e792c8ffc144b9
[INFO 24-01-24 17:08:24.2393 +04 kernel.cc:1061] Use fast generic engine


i:22


[WARNING 24-01-24 17:08:24.7243 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:24.7243 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:24.7243 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:08:25.3161 +04 kernel.cc:1233] Loading model from path /tmp/tmp_o_sdj1v/model/ with prefix 2935fd09242f42db
[INFO 24-01-24 17:08:25.3188 +04 kernel.cc:1061] Use fast generic engine


i:23


[WARNING 24-01-24 17:08:25.7872 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:25.7872 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:25.7872 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:08:26.4911 +04 kernel.cc:1233] Loading model from path /tmp/tmpkny9taxw/model/ with prefix c59e1d84eec340a4
[INFO 24-01-24 17:08:26.4964 +04 abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-01-24 17:08:26.4964 +04 kernel.cc:1061] Use fast generic engine


i:24


[WARNING 24-01-24 17:08:27.1074 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:27.1074 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:27.1074 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:08:29.2518 +04 kernel.cc:1233] Loading model from path /tmp/tmplig0icx0/model/ with prefix c7df30edef7444e8
[INFO 24-01-24 17:08:29.2653 +04 kernel.cc:1061] Use fast generic engine


i:25


[WARNING 24-01-24 17:08:31.6443 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:31.6444 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:31.6444 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:08:32.8659 +04 kernel.cc:1233] Loading model from path /tmp/tmp99swk7nz/model/ with prefix 45f55f2781a849bd
[INFO 24-01-24 17:08:32.8718 +04 kernel.cc:1061] Use fast generic engine


i:26


[WARNING 24-01-24 17:08:33.4018 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:33.4018 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:33.4018 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:08:34.3464 +04 kernel.cc:1233] Loading model from path /tmp/tmp9ywghc1v/model/ with prefix 0a371502c9a744e6
[INFO 24-01-24 17:08:34.3525 +04 kernel.cc:1061] Use fast generic engine


i:27


[WARNING 24-01-24 17:08:35.0679 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:35.0679 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:35.0679 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:08:35.9293 +04 kernel.cc:1233] Loading model from path /tmp/tmp3q585kby/model/ with prefix 4a44a38a29184d1c
[INFO 24-01-24 17:08:35.9348 +04 kernel.cc:1061] Use fast generic engine


i:28


[WARNING 24-01-24 17:08:36.5798 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:36.5799 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:36.5799 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:08:37.8169 +04 kernel.cc:1233] Loading model from path /tmp/tmp8xzs4cd5/model/ with prefix 6c06595a2c5a439a
[INFO 24-01-24 17:08:37.8248 +04 abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-01-24 17:08:37.8250 +04 kernel.cc:1061] Use fast generic engine


i:29


[WARNING 24-01-24 17:08:41.3412 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:41.3413 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:41.3413 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:08:42.6922 +04 kernel.cc:1233] Loading model from path /tmp/tmp_coxurw7/model/ with prefix 3f2453304ec24966
[INFO 24-01-24 17:08:42.7013 +04 kernel.cc:1061] Use fast generic engine


i:30


[WARNING 24-01-24 17:08:43.3010 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:43.3010 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:43.3010 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:08:44.5454 +04 kernel.cc:1233] Loading model from path /tmp/tmp72i_0bdo/model/ with prefix e6a3febac29d4d87
[INFO 24-01-24 17:08:44.5700 +04 kernel.cc:1061] Use fast generic engine


i:31


[WARNING 24-01-24 17:08:45.2275 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:45.2275 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:45.2275 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:08:46.0331 +04 kernel.cc:1233] Loading model from path /tmp/tmph2rdhifg/model/ with prefix 34fe33ea8fd845c5
[INFO 24-01-24 17:08:46.0378 +04 quick_scorer_extended.cc:903] The binary was compiled without AVX2 support, but your CPU supports it. Enable it for faster model inference.
[INFO 24-01-24 17:08:46.0384 +04 kernel.cc:1061] Use fast generic engine


i:32


[WARNING 24-01-24 17:08:46.6080 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:46.6081 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:46.6081 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:08:48.2670 +04 kernel.cc:1233] Loading model from path /tmp/tmpwi3_etjn/model/ with prefix 3190036ff2634fbd
[INFO 24-01-24 17:08:48.2801 +04 abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-01-24 17:08:48.2802 +04 kernel.cc:1061] Use fast generic engine


i:33


[WARNING 24-01-24 17:08:49.7010 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:49.7010 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:49.7010 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:08:52.1129 +04 kernel.cc:1233] Loading model from path /tmp/tmpbpjefhx_/model/ with prefix 00b29d7ff32942cd
[INFO 24-01-24 17:08:52.1593 +04 kernel.cc:1061] Use fast generic engine


i:34


[WARNING 24-01-24 17:08:52.9548 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:52.9549 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:52.9549 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:08:53.6673 +04 kernel.cc:1233] Loading model from path /tmp/tmp8bzbt_bp/model/ with prefix 9793ef80cbe94d0e
[INFO 24-01-24 17:08:53.6714 +04 kernel.cc:1061] Use fast generic engine


i:35


[WARNING 24-01-24 17:08:54.6530 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:54.6531 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:54.6531 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:08:55.9235 +04 kernel.cc:1233] Loading model from path /tmp/tmp8b6tvb0j/model/ with prefix 2663245ad5da4d70
[INFO 24-01-24 17:08:55.9319 +04 kernel.cc:1061] Use fast generic engine


i:36


[WARNING 24-01-24 17:08:56.8782 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:56.8782 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:56.8782 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:08:58.5555 +04 kernel.cc:1233] Loading model from path /tmp/tmpd_bryn49/model/ with prefix 76dad5689688499f
[INFO 24-01-24 17:08:58.5663 +04 abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-01-24 17:08:58.5663 +04 kernel.cc:1061] Use fast generic engine


i:37


[WARNING 24-01-24 17:08:59.2679 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:59.2679 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:08:59.2680 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:08:59.9669 +04 kernel.cc:1233] Loading model from path /tmp/tmpvlex4oub/model/ with prefix 3f359a25a3a04562
[INFO 24-01-24 17:08:59.9721 +04 kernel.cc:1061] Use fast generic engine


i:38


[WARNING 24-01-24 17:09:00.4802 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:00.4803 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:00.4803 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:09:01.1933 +04 kernel.cc:1233] Loading model from path /tmp/tmpovlxmiaz/model/ with prefix 54f08634f9a44115
[INFO 24-01-24 17:09:01.1994 +04 kernel.cc:1061] Use fast generic engine


i:39


[WARNING 24-01-24 17:09:01.6937 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:01.6937 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:01.6938 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:09:02.4529 +04 kernel.cc:1233] Loading model from path /tmp/tmp3slktshu/model/ with prefix 20b02b8e2f9048c1
[INFO 24-01-24 17:09:02.4603 +04 kernel.cc:1061] Use fast generic engine


i:40


[WARNING 24-01-24 17:09:02.9773 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:02.9773 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:02.9773 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:09:03.5824 +04 kernel.cc:1233] Loading model from path /tmp/tmpdknat3f5/model/ with prefix c4c3cff1cded419d
[INFO 24-01-24 17:09:03.5843 +04 kernel.cc:1061] Use fast generic engine


i:41


[WARNING 24-01-24 17:09:04.2074 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:04.2074 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:04.2075 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:09:05.6853 +04 kernel.cc:1233] Loading model from path /tmp/tmpcmepmslm/model/ with prefix cfb763f114ff43a4
[INFO 24-01-24 17:09:05.7051 +04 kernel.cc:1061] Use fast generic engine


i:42


[WARNING 24-01-24 17:09:06.8033 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:06.8033 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:06.8034 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:09:08.3001 +04 kernel.cc:1233] Loading model from path /tmp/tmpplsd7x1k/model/ with prefix 16db270d37cf46bb
[INFO 24-01-24 17:09:08.3269 +04 kernel.cc:1061] Use fast generic engine


i:43


[WARNING 24-01-24 17:09:09.6478 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:09.6479 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:09.6479 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:09:11.3367 +04 kernel.cc:1233] Loading model from path /tmp/tmplfxl_7en/model/ with prefix 781a97c1f86d4191
[INFO 24-01-24 17:09:11.3517 +04 abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-01-24 17:09:11.3518 +04 kernel.cc:1061] Use fast generic engine


i:44


[WARNING 24-01-24 17:09:12.3263 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:12.3264 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:12.3264 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:09:13.3174 +04 kernel.cc:1233] Loading model from path /tmp/tmpiah92cqz/model/ with prefix 181051fb87b5433f
[INFO 24-01-24 17:09:13.3233 +04 kernel.cc:1061] Use fast generic engine


i:45


[WARNING 24-01-24 17:09:13.8850 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:13.8850 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:13.8850 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:09:14.4445 +04 kernel.cc:1233] Loading model from path /tmp/tmp7zuux7vx/model/ with prefix 45293073342e4c23
[INFO 24-01-24 17:09:14.4461 +04 kernel.cc:1061] Use fast generic engine


i:46


[WARNING 24-01-24 17:09:14.8972 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:14.8972 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:14.8972 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:09:15.6667 +04 kernel.cc:1233] Loading model from path /tmp/tmpvmenwqp_/model/ with prefix ab18f369f104438f
[INFO 24-01-24 17:09:15.6791 +04 kernel.cc:1061] Use fast generic engine


i:47


[WARNING 24-01-24 17:09:16.4288 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:16.4288 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:16.4288 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:09:17.8493 +04 kernel.cc:1233] Loading model from path /tmp/tmpwaeqka3x/model/ with prefix 3bb6047084a6477b
[INFO 24-01-24 17:09:17.8644 +04 quick_scorer_extended.cc:903] The binary was compiled without AVX2 support, but your CPU supports it. Enable it for faster model inference.
[INFO 24-01-24 17:09:17.8653 +04 kernel.cc:1061] Use fast generic engine


i:48


[WARNING 24-01-24 17:09:18.8409 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:18.8410 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:18.8410 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:09:20.0030 +04 kernel.cc:1233] Loading model from path /tmp/tmpe2mkltg1/model/ with prefix 1c063a4357c44029
[INFO 24-01-24 17:09:20.0093 +04 kernel.cc:1061] Use fast generic engine


i:49


[WARNING 24-01-24 17:09:20.9567 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:20.9568 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:20.9568 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:09:21.5769 +04 kernel.cc:1233] Loading model from path /tmp/tmpwdew0y_3/model/ with prefix 629ff9cbed724efb
[INFO 24-01-24 17:09:21.5798 +04 abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-01-24 17:09:21.5799 +04 kernel.cc:1061] Use fast generic engine


i:50


[WARNING 24-01-24 17:09:22.0396 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:22.0396 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:22.0396 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:09:22.7054 +04 kernel.cc:1233] Loading model from path /tmp/tmpxtlsu28v/model/ with prefix 6440f85bbd204968
[INFO 24-01-24 17:09:22.7115 +04 kernel.cc:1061] Use fast generic engine


i:51


[WARNING 24-01-24 17:09:23.1400 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:23.1401 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:23.1401 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:09:23.8818 +04 kernel.cc:1233] Loading model from path /tmp/tmpltz640vy/model/ with prefix 256b1f21c2d64b97
[INFO 24-01-24 17:09:23.8896 +04 kernel.cc:1061] Use fast generic engine


i:52


[WARNING 24-01-24 17:09:24.3212 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:24.3213 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:24.3213 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:09:24.8848 +04 kernel.cc:1233] Loading model from path /tmp/tmp8qxxksfa/model/ with prefix 1317034302ea47d5
[INFO 24-01-24 17:09:24.8902 +04 kernel.cc:1061] Use fast generic engine


i:53


[WARNING 24-01-24 17:09:25.3008 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:25.3008 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:25.3008 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:09:27.1730 +04 kernel.cc:1233] Loading model from path /tmp/tmpoxpjwwv0/model/ with prefix 58e06e0ba1184366
[INFO 24-01-24 17:09:27.1821 +04 kernel.cc:1061] Use fast generic engine


i:54


[WARNING 24-01-24 17:09:28.3302 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:28.3302 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:28.3303 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:09:29.8143 +04 kernel.cc:1233] Loading model from path /tmp/tmpekldc5mb/model/ with prefix aa707b07c43947df
[INFO 24-01-24 17:09:29.8168 +04 kernel.cc:1061] Use fast generic engine


i:55


[WARNING 24-01-24 17:09:30.7163 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:30.7163 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:30.7164 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:09:31.5001 +04 kernel.cc:1233] Loading model from path /tmp/tmp78licbj4/model/ with prefix 75b50bdd3e774b6a
[INFO 24-01-24 17:09:31.5129 +04 kernel.cc:1061] Use fast generic engine


i:56


[WARNING 24-01-24 17:09:32.9266 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:32.9266 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:32.9267 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:09:33.8805 +04 kernel.cc:1233] Loading model from path /tmp/tmpgtisyng7/model/ with prefix 1da88f93f1b84d53
[INFO 24-01-24 17:09:33.8926 +04 abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-01-24 17:09:33.8927 +04 kernel.cc:1061] Use fast generic engine


i:57


[WARNING 24-01-24 17:09:34.5067 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:34.5068 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:34.5068 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:09:35.2275 +04 kernel.cc:1233] Loading model from path /tmp/tmp5i2hlj3g/model/ with prefix d1cc437a7945423d
[INFO 24-01-24 17:09:35.2299 +04 kernel.cc:1061] Use fast generic engine


i:58


[WARNING 24-01-24 17:09:36.1645 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:36.1645 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:36.1645 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:09:38.3195 +04 kernel.cc:1233] Loading model from path /tmp/tmpgxkf8fwq/model/ with prefix a333bf784b0d4a3e
[INFO 24-01-24 17:09:38.3297 +04 kernel.cc:1061] Use fast generic engine


i:59


[WARNING 24-01-24 17:09:39.8988 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:39.8989 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:39.8989 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:09:41.1909 +04 kernel.cc:1233] Loading model from path /tmp/tmpu0cxqnh7/model/ with prefix 1beda14d74a541ad
[INFO 24-01-24 17:09:41.1968 +04 kernel.cc:1061] Use fast generic engine


i:60


[WARNING 24-01-24 17:09:41.7436 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:41.7436 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:41.7436 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:09:42.5561 +04 kernel.cc:1233] Loading model from path /tmp/tmpydjyv6d8/model/ with prefix 278ef7d9c36e431d
[INFO 24-01-24 17:09:42.5613 +04 kernel.cc:1061] Use fast generic engine


i:61


[WARNING 24-01-24 17:09:43.1086 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:43.1086 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:43.1086 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:09:43.7792 +04 kernel.cc:1233] Loading model from path /tmp/tmpxz4amxv1/model/ with prefix 6ce951e718f4401c
[INFO 24-01-24 17:09:43.7819 +04 kernel.cc:1061] Use fast generic engine


i:62


[WARNING 24-01-24 17:09:44.3284 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:44.3285 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:44.3285 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:09:45.0469 +04 kernel.cc:1233] Loading model from path /tmp/tmpa9f232zq/model/ with prefix 368f744f25ff499a
[INFO 24-01-24 17:09:45.0547 +04 abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-01-24 17:09:45.0548 +04 kernel.cc:1061] Use fast generic engine


i:63


[WARNING 24-01-24 17:09:45.5046 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:45.5046 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:45.5046 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:09:46.6860 +04 kernel.cc:1233] Loading model from path /tmp/tmpq2e87wti/model/ with prefix 86f4ca54612f42c8
[INFO 24-01-24 17:09:46.6955 +04 kernel.cc:1061] Use fast generic engine


i:64


[WARNING 24-01-24 17:09:47.8233 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:47.8234 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:47.8234 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:09:49.0774 +04 kernel.cc:1233] Loading model from path /tmp/tmp7gdbyzeo/model/ with prefix 7f385a710b2e4d3d
[INFO 24-01-24 17:09:49.0833 +04 quick_scorer_extended.cc:903] The binary was compiled without AVX2 support, but your CPU supports it. Enable it for faster model inference.
[INFO 24-01-24 17:09:49.0844 +04 kernel.cc:1061] Use fast generic engine


i:65


[WARNING 24-01-24 17:09:50.0114 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:50.0114 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:50.0114 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:09:50.8774 +04 kernel.cc:1233] Loading model from path /tmp/tmp7q1q4xhu/model/ with prefix dce779849c444d13
[INFO 24-01-24 17:09:50.8794 +04 kernel.cc:1061] Use fast generic engine


i:66


[WARNING 24-01-24 17:09:51.3123 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:51.3123 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:51.3123 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:09:51.8927 +04 kernel.cc:1233] Loading model from path /tmp/tmpmiu_h574/model/ with prefix 3edb0ef76cd34fa9
[INFO 24-01-24 17:09:51.8955 +04 kernel.cc:1061] Use fast generic engine


i:67


[WARNING 24-01-24 17:09:52.3126 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:52.3126 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:52.3126 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:09:53.0774 +04 kernel.cc:1233] Loading model from path /tmp/tmpk_vglsb0/model/ with prefix 996f76af793246ae
[INFO 24-01-24 17:09:53.0889 +04 kernel.cc:1061] Use fast generic engine


i:68


[WARNING 24-01-24 17:09:53.5128 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:53.5128 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:53.5128 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:09:54.1743 +04 kernel.cc:1233] Loading model from path /tmp/tmpmt5a753l/model/ with prefix 552ee2983ae74272
[INFO 24-01-24 17:09:54.1792 +04 kernel.cc:1061] Use fast generic engine


i:69


[WARNING 24-01-24 17:09:54.6271 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:54.6271 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:54.6271 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:09:55.3072 +04 kernel.cc:1233] Loading model from path /tmp/tmpp6spff2n/model/ with prefix e28d6551024149a5
[INFO 24-01-24 17:09:55.3103 +04 abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-01-24 17:09:55.3104 +04 kernel.cc:1061] Use fast generic engine


i:70


[WARNING 24-01-24 17:09:55.9321 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:55.9321 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:55.9321 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:09:57.2766 +04 kernel.cc:1233] Loading model from path /tmp/tmp2avc3hdw/model/ with prefix b96efd84a1c045d4
[INFO 24-01-24 17:09:57.2855 +04 kernel.cc:1061] Use fast generic engine


i:71


[WARNING 24-01-24 17:09:58.2050 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:58.2051 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:09:58.2051 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:09:59.4124 +04 kernel.cc:1233] Loading model from path /tmp/tmpb0vyu23s/model/ with prefix 1c576b1d2cd544ad
[INFO 24-01-24 17:09:59.4229 +04 kernel.cc:1061] Use fast generic engine


i:72


[WARNING 24-01-24 17:10:00.4314 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:00.4314 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:00.4314 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:10:01.2451 +04 kernel.cc:1233] Loading model from path /tmp/tmp3316s4cj/model/ with prefix 74a6febde66f44ad
[INFO 24-01-24 17:10:01.2560 +04 kernel.cc:1061] Use fast generic engine


i:73


[WARNING 24-01-24 17:10:01.7397 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:01.7397 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:01.7397 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:10:02.3178 +04 kernel.cc:1233] Loading model from path /tmp/tmpu5zpcdcg/model/ with prefix 9ef2834954c24628
[INFO 24-01-24 17:10:02.3209 +04 kernel.cc:1061] Use fast generic engine


i:74


[WARNING 24-01-24 17:10:02.7697 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:02.7697 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:02.7697 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:10:03.4445 +04 kernel.cc:1233] Loading model from path /tmp/tmp1ap_q535/model/ with prefix fb4bb3be42fa463e
[INFO 24-01-24 17:10:03.4501 +04 kernel.cc:1061] Use fast generic engine


i:75


[WARNING 24-01-24 17:10:03.8792 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:03.8792 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:03.8792 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:10:04.6110 +04 kernel.cc:1233] Loading model from path /tmp/tmpdqw389l5/model/ with prefix 80a38069bc1c4966
[INFO 24-01-24 17:10:04.6161 +04 kernel.cc:1061] Use fast generic engine


i:76


[WARNING 24-01-24 17:10:05.0384 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:05.0385 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:05.0385 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:10:05.7063 +04 kernel.cc:1233] Loading model from path /tmp/tmphrd75ryh/model/ with prefix e56c29ea1b584988
[INFO 24-01-24 17:10:05.7093 +04 abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-01-24 17:10:05.7093 +04 kernel.cc:1061] Use fast generic engine


i:77


[WARNING 24-01-24 17:10:06.6240 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:06.6241 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:06.6241 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:10:07.8160 +04 kernel.cc:1233] Loading model from path /tmp/tmp9gftcfye/model/ with prefix b7de8b36eec143a5
[INFO 24-01-24 17:10:07.8215 +04 kernel.cc:1061] Use fast generic engine


i:78


[WARNING 24-01-24 17:10:08.7858 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:08.7859 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:08.7859 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:10:10.0669 +04 kernel.cc:1233] Loading model from path /tmp/tmp2oyxhi33/model/ with prefix 0c247fbaa9bf4fd9
[INFO 24-01-24 17:10:10.0745 +04 kernel.cc:1061] Use fast generic engine


i:79


[WARNING 24-01-24 17:10:10.7788 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:10.7789 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:10.7790 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:10:11.3875 +04 kernel.cc:1233] Loading model from path /tmp/tmpg32fmpjb/model/ with prefix 707482c018bb4791
[INFO 24-01-24 17:10:11.3909 +04 kernel.cc:1061] Use fast generic engine


i:80


[WARNING 24-01-24 17:10:11.8252 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:11.8252 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:11.8252 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:10:12.4718 +04 kernel.cc:1233] Loading model from path /tmp/tmpc69g4n6h/model/ with prefix 6366d22766bd4fe9
[INFO 24-01-24 17:10:12.4761 +04 kernel.cc:1061] Use fast generic engine


i:81


[WARNING 24-01-24 17:10:12.9241 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:12.9242 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:12.9242 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:10:13.5940 +04 kernel.cc:1233] Loading model from path /tmp/tmpkcg1r_c1/model/ with prefix fcb725142ec544b9
[INFO 24-01-24 17:10:13.5999 +04 kernel.cc:1061] Use fast generic engine


i:82


[WARNING 24-01-24 17:10:14.1504 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:14.1505 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:14.1506 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:10:14.8063 +04 kernel.cc:1233] Loading model from path /tmp/tmpbkoxp62i/model/ with prefix 3d732bd9819e4325
[INFO 24-01-24 17:10:14.8100 +04 kernel.cc:1061] Use fast generic engine


i:83


[WARNING 24-01-24 17:10:15.2441 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:15.2442 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:15.2443 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:10:16.0063 +04 kernel.cc:1233] Loading model from path /tmp/tmpt4krfykp/model/ with prefix 99c56d54dbfa4426
[INFO 24-01-24 17:10:16.0098 +04 abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-01-24 17:10:16.0098 +04 kernel.cc:1061] Use fast generic engine


i:84


[WARNING 24-01-24 17:10:16.4543 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:16.4543 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:16.4543 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:10:17.1951 +04 kernel.cc:1233] Loading model from path /tmp/tmpiozlqt0z/model/ with prefix 419e7b5f9bec474e
[INFO 24-01-24 17:10:17.2046 +04 kernel.cc:1061] Use fast generic engine


i:85


[WARNING 24-01-24 17:10:17.6427 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:17.6427 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:17.6428 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:10:19.1229 +04 kernel.cc:1233] Loading model from path /tmp/tmpleokkeo_/model/ with prefix 456cc972cd134324
[INFO 24-01-24 17:10:19.1267 +04 quick_scorer_extended.cc:903] The binary was compiled without AVX2 support, but your CPU supports it. Enable it for faster model inference.
[INFO 24-01-24 17:10:19.1274 +04 kernel.cc:1061] Use fast generic engine


i:86


[WARNING 24-01-24 17:10:19.6932 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:19.6933 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:19.6933 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:10:20.4995 +04 kernel.cc:1233] Loading model from path /tmp/tmpd3sfi2eg/model/ with prefix 615e920e77324fff
[INFO 24-01-24 17:10:20.5090 +04 kernel.cc:1061] Use fast generic engine


i:87


[WARNING 24-01-24 17:10:21.0323 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:21.0323 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:21.0323 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:10:21.8302 +04 kernel.cc:1233] Loading model from path /tmp/tmprpmao5_9/model/ with prefix 65a2e713b7104750
[INFO 24-01-24 17:10:21.8407 +04 kernel.cc:1061] Use fast generic engine


i:88


[WARNING 24-01-24 17:10:22.3388 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:22.3389 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:22.3389 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:10:23.1353 +04 kernel.cc:1233] Loading model from path /tmp/tmpt540c4l_/model/ with prefix edd7421012fa4ab5
[INFO 24-01-24 17:10:23.1411 +04 kernel.cc:1061] Use fast generic engine


i:89


[WARNING 24-01-24 17:10:23.6150 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:23.6150 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:23.6150 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:10:24.2433 +04 kernel.cc:1233] Loading model from path /tmp/tmpm7ldrupu/model/ with prefix a2b75986e1d647dd
[INFO 24-01-24 17:10:24.2481 +04 kernel.cc:1061] Use fast generic engine


i:90


[WARNING 24-01-24 17:10:24.7149 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:24.7150 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:24.7150 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:10:25.3808 +04 kernel.cc:1233] Loading model from path /tmp/tmpde5g0lja/model/ with prefix d4c4309cff73431e
[INFO 24-01-24 17:10:25.3872 +04 kernel.cc:1061] Use fast generic engine


i:91


[WARNING 24-01-24 17:10:26.3384 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:26.3384 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:26.3384 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:10:27.5907 +04 kernel.cc:1233] Loading model from path /tmp/tmp__9dk1u5/model/ with prefix df3998f7318a4de5
[INFO 24-01-24 17:10:27.6026 +04 abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-01-24 17:10:27.6028 +04 kernel.cc:1061] Use fast generic engine


i:92


[WARNING 24-01-24 17:10:28.6393 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:28.6393 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:28.6393 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:10:30.3038 +04 kernel.cc:1233] Loading model from path /tmp/tmpfw1lgn_g/model/ with prefix 0e9a546f384a4c66
[INFO 24-01-24 17:10:30.3218 +04 kernel.cc:1061] Use fast generic engine


i:93


[WARNING 24-01-24 17:10:30.8869 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:30.8869 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:30.8869 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:10:31.6913 +04 kernel.cc:1233] Loading model from path /tmp/tmp92to0ebm/model/ with prefix 77d53599a08747fa
[INFO 24-01-24 17:10:31.6980 +04 kernel.cc:1061] Use fast generic engine


i:94


[WARNING 24-01-24 17:10:32.1571 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:32.1571 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:32.1571 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:10:32.7627 +04 kernel.cc:1233] Loading model from path /tmp/tmpslqh_w2l/model/ with prefix cfa46a6eac2645a8
[INFO 24-01-24 17:10:32.7671 +04 kernel.cc:1061] Use fast generic engine


i:95


[WARNING 24-01-24 17:10:33.2345 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:33.2345 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:33.2346 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:10:34.0185 +04 kernel.cc:1233] Loading model from path /tmp/tmps_bcd84s/model/ with prefix e1cded0017af4888
[INFO 24-01-24 17:10:34.0230 +04 kernel.cc:1061] Use fast generic engine


i:96


[WARNING 24-01-24 17:10:34.5244 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:34.5244 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:34.5244 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:10:35.2014 +04 kernel.cc:1233] Loading model from path /tmp/tmpoh12uiny/model/ with prefix 3f74840104d94cac
[INFO 24-01-24 17:10:35.2062 +04 kernel.cc:1061] Use fast generic engine


i:97


[WARNING 24-01-24 17:10:35.7310 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:35.7311 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:35.7311 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:10:36.9505 +04 kernel.cc:1233] Loading model from path /tmp/tmphjcrzfjn/model/ with prefix 5d2f394406114b42
[INFO 24-01-24 17:10:36.9567 +04 kernel.cc:1061] Use fast generic engine


i:98


[WARNING 24-01-24 17:10:38.3079 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:38.3080 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:38.3080 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:10:39.8227 +04 kernel.cc:1233] Loading model from path /tmp/tmpi2lrtts2/model/ with prefix 57de033ea7274c6d
[INFO 24-01-24 17:10:39.8314 +04 abstract_model.cc:1344] Engine "GradientBoostedTreesQuickScorerExtended" built
[INFO 24-01-24 17:10:39.8315 +04 kernel.cc:1061] Use fast generic engine


i:99


[WARNING 24-01-24 17:10:40.6787 +04 gradient_boosted_trees.cc:1886] "goss_alpha" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:40.6788 +04 gradient_boosted_trees.cc:1897] "goss_beta" set but "sampling_method" not equal to "GOSS".
[WARNING 24-01-24 17:10:40.6788 +04 gradient_boosted_trees.cc:1911] "selective_gradient_boosting_ratio" set but "sampling_method" not equal to "SELGB".
[INFO 24-01-24 17:10:41.4328 +04 kernel.cc:1233] Loading model from path /tmp/tmpzfqnf_1g/model/ with prefix 8865d408eaad4c3c
[INFO 24-01-24 17:10:41.4410 +04 kernel.cc:1061] Use fast generic engine


Submission exported to data/tf_lead_submission.csv


# What is next

If you want to learn more about TensorFlow Decision Forests and its advanced features, you can follow the official documentation [here](https://www.tensorflow.org/decision_forests)